hack/order-prow-job-config.sh

```diff
@@ -15,8 +15,9 @@ if [[ ! -d "${base_dir}" ]]; then
 fi
 
 ci_operator_dir="${base_dir}/ci-operator"
+config_path="${base_dir}/core-services/sanitize-prow-jobs/_config.yaml"
```

Comment:
This is used in once place -- just add it to the `--config-path` line, no need for a new var

Reply:
Done.